### PR TITLE
Update test

### PR DESCRIPTION
**Pull request by janisimonen**
Factory reset and update tests

### CODE REVIEW

**Comment by JammuKekkonen** [comment]
Should you take the internal PR-25 also into this change set?

File: pelion_test_lib/cloud/libraries/update.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
+"""
+Copyright 2019 ARM Limited
```
Comment (teetak01) [inline pelion_test_lib/cloud/libraries/update.py]: 2020 ?

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+manifest-tool
```
Comment (teetak01) [inline requirements.txt]: for next version this should be fixed to only accept major-version 1, if there is version 2, it should not automatically try to use it.

Comment (teetak01) [reply requirements.txt]: It might be better to try to fix limit the versions of other dependencies at least a bit.

**Pull request merged**